### ISSUE TITLE
importing ProfileReport seems to interfere with pandas plotting

### ISSUE THREAD

**Issue by raybellwaves** [pandas-profiling/pandas-profiling#837]
**Describe the bug**

Running
```
from pandas_profiling import ProfileReport
import pandas as pd
```
in the same notebook cells seems to interfere with pandas plotting

**To Reproduce**

Run this as a cell in jupyter lab (may be same as jupyter notebook)

```
import pandas as pd
from pandas_profiling import ProfileReport

df = pd.DataFrame(data={"col1": [1, 2]})
df.plot()
```

<img width="507" alt="Screen Shot 2021-09-28 at 5 02 21 PM" src="https://user-images.githubusercontent.com/17162724/135165192-00755b6e-b275-436e-92f1-0958059b1903.png">

<img width="503" alt="Screen Shot 2021-09-28 at 5 03 39 PM" src="https://user-images.githubusercontent.com/17162724/135165389-0489d2a0-d6bf-418f-b0a5-f784fef759e2.png">

**Version information:**

pandas-profiling          3.1.0

**Comment by sbrugman** [comment]
Good catch. Might have to do with the setting of the backend in 3.1.0. 

Two ways of checking this: check if the same happens with 3.0.0 or set the backend of matplotlib after importing PP.

**Comment by raybellwaves** [comment]
I can confirm the same does not happen on 3.0.0. An easy place to verify is a binder link e.g.
https://mybinder.org/v2/gh/scikit-learn/scikit-learn/1.0.X?urlpath=lab/tree/notebooks/auto_examples/release_highlights/plot_release_highlights_1_0_0.ipynb

`!conda install -c conda-forge pandas-profiling --y`

```
import pandas as pd
from pandas_profiling import ProfileReport

df = pd.DataFrame(data={"col1": [1, 2]})
df.plot()
```

<img width="1035" alt="Screen Shot 2021-10-10 at 9 04 40 PM" src="https://user-images.githubusercontent.com/17162724/136719898-2147ff76-68c8-4639-9565-19813665ee6b.png">
